content/en-us/studio/plugins.md

```diff
@@ -9,6 +9,13 @@ A **plugin** is an extension that adds additional features or functionality to S
 
 You can create your own plugins to improve your workflow in Studio. The following code sample is a plugin called **EmptyScriptAdder** that inserts an empty script as the child of an object or in `Class.ServerScriptService`. The following sections explain the major parts to creating this plugin.
 
+To start, you should enable Plugin Debugging in the Studio settings, this will
+expose the `Class.PluginDebugService` in Studio which provides real-time debugging
+for your plugin's code, as well as making it easier to reload and save your
+plugin.
+
+<img src="../assets/studio/plugins/Plugin-EnableDebugging.png" width="320"/>
```

Comment:
```suggestion
To begin, you should enable **Plugin Debugging Enabled** in the **Studio** section of Studio's settings. This will expose the `Class.PluginDebugService` in Studio which provides real-time debugging for your plugin's code and makes it easier to reload and save your plugin.
```